### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by lslezak**
- More unit tests
- Bump version to 4.4.0, this should be the first public release

### CODE REVIEW

File: package/yast2-migration-sle.changes [hunk 1]
```diff
@@ -1,3 +1,14 @@
+-------------------------------------------------------------------
+Fri Mar 25 15:20:58 UTC 2022 - Ladislav Slezák <lslezak@suse.cz>
+
+- Updated README
+- Fixed starting the module from the control center
+- Allow migration to SLED or other products if supported by SCC
```
Comment (dgdavid) [inline package/yast2-migration-sle.changes:6]: SLED or SLE?

File: test/main_workflow_test.rb [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        allow(Migration::Restarter.instance).to receive(:restarted).and_return(false)
+      end
+
+      context "base product is not found" do
+        before do
+          expect(Registration::SwMgmt).to receive(:find_base_product).and_return(nil)
+          allow(Registration::Helpers).to receive(:report_no_base_product)
+        end
+
+        it "displays an error message if base product is not found" do
```
Comment (dgdavid) [inline test/main_workflow_test.rb]:
```suggestion
        it "displays an error message" do
```

The second part is already stated in the context.

File: test/main_workflow_test.rb [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+          expect(Registration::Helpers).to receive(:report_no_base_product)
+          subject.run
+        end
+
+        it "returns :abort" do
+          expect(subject.run).to eq(:abort)
+        end
+      end
+
+      # TODO: test the real migration case
```
Comment (dgdavid) [inline test/main_workflow_test.rb:70]: Leftover? Or it is there on purpose?

**Review by dgdavid — APPROVED** [review]

File: test/main_workflow_test.rb (hunk 3 above)
Comment (lslezak) [reply test/main_workflow_test.rb:70]: On purpose, as a reminder that it is not complete. :smiley:

File: test/main_workflow_test.rb (hunk 2 above)
Comment (lslezak) [reply test/main_workflow_test.rb]: Ah, I created the context later without updating the description...

File: package/yast2-migration-sle.changes (hunk 1 above)
Comment (lslezak) [reply package/yast2-migration-sle.changes:6]: SLED, SLE is a generic term for whole family.

**Pull request merged**